src/allocator.f90

```diff
@@ -1,37 +1,37 @@
-module m_allocator
-   use iso_fortran_env, only: stderr => error_unit
-
-   use m_common, only: dp, DIR_X, DIR_Y, DIR_Z, DIR_C
-
-   implicit none
-
-   type :: allocator_t
-     !! An instance of type allocator_t is responsible for the
-     !! maintenance of a linked list of instances of equal size
-     !! [[m_allocator(module):field_t(type)]] objects:
-     !!
-     !! ```
-     !!       ---- ---- ----     ---- ---- ----
-     !! ...-->|id=1|data|next|-->|id=0|data|next|-->null()
-     !!       ---- ---- ----     ---- ---- ----
-     !! ```
-     !!
-     !! the last block's `next` pointer being non associated.
-     !!
-     !! User code can request access to a memory block by using the
-     !! type bound procedure
-     !! [[m_allocator(module):get_block(function)]].  If the list is
-     !! not empty, a pointer to the first block on the list is
-     !! returned and the block is detached from the list.  If the list
-     !! is empty (i.e. all initially allocated blocks are currently
-     !! referenced to) then a new block is allocated before a pointer
-     !! to it is returned.
-     !!
-     !! In order to reuse memory it is important that user code
-     !! release blocks when they are not needed anymore.  This is done
-     !! by calling the type bound procedure
-     !! [[m_allocator(module):release_block(subroutine)]].  The
-     !! released block is then pushed in front of the block list.
+  module m_allocator
```

Comment:
Commented on the wrong side and its actually not visible due to that. I mean the 2 spaces before `module m_allocator`

Reply:
see https://github.com/xcompact3d/x3d2/pull/80#discussion_r1539124115